modules/ocf/templates/networking/interfaces.erb

```diff
@@ -16,3 +16,8 @@ iface <%= @iface %> inet6 static
   address <%= @ipaddress6 %>
   netmask <%= @netmask6 %>
   gateway <%= @gateway6 %>
+
+<%- if @lsbdistid == 'Raspbian' -%>
+pre-up wpa_supplicant -B -Dnl80211 -i<%= @iface %> -c /etc/wpa_supplicant/wpa_supplicant.conf -f /var/log/wpa_supplicant.log
```

Comment:
I should probably include `/etc/wpa_supplicant/wpa_supplicant.conf` in here pulling the password from the private share.